### DISCUSSION TITLE
What is WebRTC media server at Ant-Media-Server?

### DISCUSSION THREAD

**Discussion by SangukChoi** [ant-media/Ant-Media-Server#4882] [FAQs] [answered]
I'm confused, could you help me?

I saw below documents.
* https://antmedia.io/docs/guides/publish-live-stream/WebRTC-WebSocket-Messaging-Reference/

After reading that document, I think that ant-media-server is signaling server when peer communicate with WebRTC.
Is it right? Is there anything that anti-media-server does regarding media transmission except signaling?

On the other hand, below documents describe WebRTC media server. 
* https://antmedia.io/what-is-webrtc-and-how-webrtc-works/

If ant-media-server is also WebRTC media server or ant-media-server do something when media stream transmission, I want to know role of ant-media-server (WebRTC media server) at the transmission of media stream.

**Comment by Mohit-3196** [comment, marked as answer]
Hi @SangukChoi ,
How are you?

I'll try to answer these questions for you.

- For peer-to-peer communication, Ant Media Server connects two peers directly.
Ant Media Server is only a signaling server to establish the connection. Video and Audio are not relayed through Ant Media Server.

- To answer the other question, Ant Media Server is a real-time streaming engine software that provides adaptive, ultra-low latency streaming by using WebRTC technology with ~0.5 seconds latency.
Ant Media Server is highly scalable both horizontally and vertically. It can run on-premise or on-cloud.
AMS supports publish and play protocols like WebRTC, RTMP, SRT, RTSP, RTMPS, MP4, HLS, DASH/CMAF, IP cameras, and other stream sources.
AMS can work both in SFU as well as MCU mode. It can be used to transcode streams and supports recording on the server side. AMS gives a lot of flexibility to users in terms of security, REST APIs, restreaming, and supports different codes like H.264, VP8, H.265, etc.
There is also a lot of SDK support like native SDKs for Android and iOS, Flutter SDK, React Native SDK, and Unity SDK, and the server can be scaled in real-time to match the requirements.

AMS used the original WebRTC that chromes uses under the hood. WebRTC is evolving fast and it means it is fast to integrate the new features of the WebRTC into Ant Media Server.
It uses ffmpeg under the hood as a library. It means we can extend Ant Media Server easily with the formats/codecs.

We focus on the product and we’re creating an ecosystem with partners and community. We have also launched the AMS marketplace to let the developers and contributors extend the Ant Media Server with the vision to be the “streaming engine of the world”.

Thank you.
